src/IpScanner/Program.cs

```diff
@@ -5,18 +5,26 @@ namespace IpScanner;
 
 internal static class Program
 {
-    private static bool IsDisplayUnknownHost => false; // TODO: add ability for change from args
+    private static bool IsDisplayUnknownHost = false;
 
     private static Action<string?> Print { get; } = Console.WriteLine;
 
     static void Main(string[] args)
     {
         try
         {
-            if (args.Length != 2)
+            if (args.Length <= 2)
                 throw new Exception("Need two arguments: <Start IP> <End IP>");
 
-            var startIp = IPAddress.Parse(args[0]);
+			if (args.Length == 3)
+			{
+				var isValidFlag = char.TryParse(args[2], out var isDisplayUnknownHost);
```

Comment:
you can simplify this part, just check:
```cs
IsDisplayUnknownHost = args[2] == "e";
```

Reply:
so what happened if their is no third parameter so what would happened if some one want to use you version